src/persist/src/s3.rs

```diff
@@ -98,13 +97,16 @@ impl S3BlobConfig {
         // is released. it contains a much more straightforward and robust way of thinking about
         // AWS client timeouts.
```

Comment:
and a small nit that this is done now 😁

Reply:
Oh, great! I wasn't sure if there was more that you wanted to do with the new version of the SDK.